### PR TITLE
Updated C28751, C30029, C30030

### PR DESCRIPTION
**Pull request by MugBergerFries**
Updated to the new format

### CODE REVIEW

**Review by DOMARS — APPROVED** [review]
@MugBergerFries - This looks good, let me know when review is complete, and I can merge in.

**Review by corob-msft — APPROVED** [review]
LGTM.

**Review by Rastaban — APPROVED** [review]

**Comment by MugBergerFries** [comment]
Re-requesting reviews as I've grouped other file changes in this PR to minimize PR clutter. These will be the only 3 files in this PR. Going forward, I will group these doc changes in batches of 5

File: windows-driver-docs-pr/devtest/28751-banned-exallocatepool-usage.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 
-**Warning C28751: Banned API Usage ExAllocatePool (BANNED_API_USAGE_EXALLOCATEPOOL)**\
-Example output: ```Banned usage of ExAllocatePool and its variants: ExAllocatePool is a banned API for improved error handling purposes.```
+## Remarks
 
-This warning indicates that a function is being used that has been banned and has a more robust or secure replacement. This specific warning indicates the use of ExAllocatePool or one of its variants, which are deprecated. See [Updating Deprecated ExAllocatePool Calls to ExAllocatePool2 and ExAllocatePool3](../kernel/updating-deprecated-exallocatepool-calls.md) for further information. A list of all banned functions covered by this error and recommended replacements can be found below: 
+See [Updating Deprecated ExAllocatePool Calls to ExAllocatePool2 and ExAllocatePool3](../kernel/updating-deprecated-exallocatepool-calls.md) for further information. A list of all banned functions covered by this error and recommended replacements can be found below.
+
+Code analysis name: BANNED_API_USAGE_EXALLOCATEPOO
```
Comment (Rastaban) [inline windows-driver-docs-pr/devtest/28751-banned-exallocatepool-usage.md:17]: I think this is missing an L ```BANNED_API_USAGE_EXALLOCATEPOOL```

Comment (MugBergerFries) [reply windows-driver-docs-pr/devtest/28751-banned-exallocatepool-usage.md]: Fixed!

**Review by Rastaban — APPROVED** [review]

**Review by corob-msft — APPROVED** [review]

**Review by MichaelSquires — APPROVED** [review]

**Comment by MugBergerFries** [comment]
@DOMARS Good to go!

**Pull request merged**